### PR TITLE
Fix PHP code style in multiple directories

### PR DESCRIPTION
**Pull request by stewartmuhanuzi**
This is a minor pull request that
- Adds 2 code style rules
- Fixes the code styles in the `config`, `database`, `resources`, `routes`, `tests` directories.

The project is properly linted with these rules.

### CODE REVIEW

File: pint.json [hunk 1]
```diff
@@ -6,6 +6,10 @@
         "new_with_braces": {
             "anonymous_class": false,
             "named_class": false
-        }
+        },
+        "ordered_imports": {
+            "sort_algorithm": "length"
+        },
+        "no_unused_imports":true
     }
```
Comment (stewartmuhanuzi) [inline pint.json:14]: These two rules ensure that the imports are ordered based on their length and unused imports will be removed automatically when you run the linter

File: pint.json [hunk 2]
```diff
@@ -6,6 +6,10 @@
         "new_with_braces": {
             "anonymous_class": false,
             "named_class": false
-        }
+        },
+        "ordered_imports": {
+            "sort_algorithm": "length"
```
Comment (stewartmuhanuzi) [inline pint.json]: This rule sorts our imports on length. No big deal here but I prefer having imports aligned based on their length.

File: pint.json [hunk 3]
```diff
@@ -6,6 +6,10 @@
...
+        },
+        "no_unused_imports":true
```
Comment (stewartmuhanuzi) [inline pint.json]: This will cleanup unused imports

**Comment by whoami15** [comment]
Thanks for this @stewartmuhanuzi, I'll update the pint config as what was discussed in Discord.

**Pull request merged**